test/Config.hs

```diff
@@ -0,0 +1,14 @@
+module Config (testConfig) where
+
+import Test.Hspec
+import Crocodealer.Config
+import Crocodealer.Core.Label()
+
+testConfig :: Config -> SpecWith ()
+testConfig config = 
+    describe "Config.loadConfig" $ 
+        it "Should load the sample configuration correctly" $ do
+            configUsername config `shouldBe` Just "my-user"
+            configRepository config `shouldBe` Just "my-org/my-repo"
+            configIgnoredRepositories config `shouldBe` [Repo {unRepo = "my-ignored-repo"},Repo {unRepo = "my-other-ignored-repo"}]
+            -- configLabelRules config `shouldBe` [Create (Label {labelName = Name {unName = "label1"}, labelDescription = "a cool label", labelColor = HexColor {unHexColor = "#000000"}}),Override (Name {unName = "old label name"}) (Name {unName = "new label name"})]
```

Comment:
I think I have a last question here. When the comment is removed the test breaks because: 
`• No instance for (Eq LabelRule) arising from a use of ‘shouldBe’`

Simply adding `Eq` to `deriving` from the `LabelRule` doesn't seem to do the trick.
What do I need to consider here?

Reply:
What error do you see after adding `Eq` to derivings for `LabelRule`? According to this error message, this should do the trick

Reply:
```
/home/maex/projects/contrib/crocodealer/src/Crocodealer/Core/Label.hs:61:21: error:
    • No instance for (Eq (Name Label))
        arising from the first field of ‘Override’ (type ‘Name Label’)
      Possible fix:
        use a standalone 'deriving instance' declaration,
          so you can specify the instance context yourself
    • When deriving the instance for (Eq LabelRule)
```

I've tried a bit of googling but the results didn't seem promising so I thought I'm heading the wrong way.

Reply:
It's done, I just needed to derive `Eq` for some more types. 

Thanks for helping :)